bitnami/postgresql/11/debian-11/rootfs/opt/bitnami/scripts/libpostgresql.sh

```diff
@@ -620,6 +620,9 @@ postgresql_initialize() {
     # Configure port
     postgresql_set_property "port" "$POSTGRESQL_PORT_NUMBER"
     is_empty_value "$POSTGRESQL_DEFAULT_TOAST_COMPRESSION" || postgresql_set_property "default_toast_compression" "$POSTGRESQL_DEFAULT_TOAST_COMPRESSION"
+    if [[ -n "$POSTGRESQL_PASSWORD_ENCRYPTION" ]]; then
+        postgresql_set_property "password_encryption" "$POSTGRESQL_PASSWORD_ENCRYPTION"
+    fi
```

Comment:
Following the same syntax used by `$POSTGRESQL_DEFAULT_TOAST_COMPRESSION`, what do you think about changing this to:

```suggestion
    is_empty_value "$POSTGRESQL_PASSWORD_ENCRYPTION" || postgresql_set_property "password_encryption" "$POSTGRESQL_PASSWORD_ENCRYPTION"
```

Besides that, could you please add `$POSTGRESQL_PASSWORD_ENCRYPTION` to the `postgresql-env.sh` files? You can use [this PR](https://github.com/bitnami/containers/pull/38520/files#diff-d95a9000323413aebbbf2f79e3842d1d1239e8d4347b6502deac418e8af088c9) as an example.

Reply:
Just did that.. Pls let me know if that works.. :pray: